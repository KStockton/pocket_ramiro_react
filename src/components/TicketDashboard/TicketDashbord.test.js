import React from 'react';
import { shallow } from 'enzyme'
import { TicketDashboard, mapDispatchToProps } from './';

describe('TicketDashboard', () => {
  let wrapper;
  let mockLocation;

  beforeEach(() => {
    mockLocation = {pathname: 'tickets'}
    wrapper = shallow(<TicketDashboard location={mockLocation}/>)
  });

  it('should match the snapshot', () => {
    expect(wrapper).toMatchSnapshot()
  });

  it('should match the snapshot if location is assets', () => {
    mockLocation.pathname = 'assets';
    wrapper = shallow(<TicketDashboard location={mockLocation}/>);
    expect(wrapper).toMatchSnapshot();
  });
});

describe('MapDispatchToProps', () => {
  let mockDispatch, mappedProps;

  beforeEach(() => {
    mockDispatch = jest.fn()
    mappedProps = mapDispatchToProps(mockDispatch)
  });

});