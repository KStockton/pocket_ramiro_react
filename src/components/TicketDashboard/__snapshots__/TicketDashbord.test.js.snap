// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`TicketDashboard should match the snapshot 1`] = `
<main>
  <Connect(CardContainer)
    dataKey="tickets"
  />
</main>
`;

exports[`TicketDashboard should match the snapshot if location is assets 1`] = `
<main>
  <Connect(CardContainer)
    dataKey="assets"
  />
</main>
`;
